Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py

```diff
@@ -413,6 +458,92 @@ def secmgr_secrule_search_command(client: Client, args: Dict[str, Any]):
     )
 
 
+def get_paged_all_collectors(client: Client, auth_token: str, payload: Dict[str, Any]) -> List:
+    """Make subsequent requests using client and other arguments
+
+    Args:
+        client (Client): `Client` class object
+        auth_token (str): authentication token to use
+        payload (Dict[str, Any]): parameter payload to use
+
+    Returns:
+        (List[Dict[str, Any]]): results list
+    """
+    result = list()
+    response = client.get_paged_all_collectors(auth_token, payload)
+    total_pages = response.get("total", 0) // payload.get("pageSize")
+
+    result.extend(response.get("results", list()))
+
+    while payload.get("page") < total_pages:  # NOTE: Check if we can implement async here
+        payload["page"] += 1
+        response = client.get_paged_all_collectors(auth_token, payload)
+        result.extend(response.get("results", list()))
+
+    return result
+
+
+def collector_get_all_command(client: Client, args: Dict[str, Any]):
+    """List all the collectors in the inventory
+
+    Args:
+        client (Client): `Client` class object
+        args (Dict[str, Any]): demisto arguments passed
+    """
+    auth_token_cmd_result = authenticate_command(client)
+    auth_token = auth_token_cmd_result.outputs
+
+    page_size = 1 if int(args.get("pageSize", 10)) < 1 else int(args.get("pageSize", 10))
+    payload = dict(
+        pageSize=page_size,
+        page=int(args.get("page", 0)),
+    )
+    results = get_paged_all_collectors(client, auth_token, payload)
+
+    return CommandResults(
+        outputs_prefix="FireMonSecurityManager.Collector",
+        outputs_key_field="id",
+        outputs=results,
+        readable_output=tableToMarkdown(
+            name="FireMon Collector:",
+            t=results,
+            removeNull=True,
+            headerTransform=pascalToSpace,
+        ),
+        raw_response=results,
+    )
+
+
+def collector_get_status_byid_command(client: Client, args: Dict[str, Any]):
+    """Get collector status by ID
+
+    Args:
+        client (Client): `Client` class object
+        args (Dict[str, Any]): demisto arguments passed
+    """
+    auth_token_cmd_result = authenticate_command(client)
+    auth_token = auth_token_cmd_result.outputs
+
+    collector_id = int(args.get("id", 0))
+    payload = dict(
+        id=collector_id,
+    )
+    results = client.get_collector_status_byid(auth_token, payload)
+
+    return CommandResults(
+        outputs_prefix="FireMonSecurityManager.Collector",
```

Comment:
```suggestion
        outputs_prefix="FireMonSecurityManager.CollectorStatus",
```
The outputs_prefix for `firemon-collector-get-status-byid` command is similar to `firemon-collector-get-all`